subsys/bootloader/cmake/provision_hex.cmake

```diff
@@ -27,9 +27,18 @@ if (CONFIG_NCS_IS_VARIANT_IMAGE)
   return()
 endif()
 
-# Build and include hex file containing provisioned data for the bootloader.
-set(PROVISION_HEX_NAME     provision.hex)
-set(PROVISION_HEX          ${PROJECT_BINARY_DIR}/${PROVISION_HEX_NAME})
+
+
+if (CONFIG_PROVISION_BUILD_STRATEGY_USE_HEX_FILE)
+  set(PROVISION_HEX ${CONFIG_PROVISION_HEX_FILE})
+  set(PROVISION_HEX_EXISTS TRUE)
+elseif(CONFIG_PROVISION_BUILD_STRATEGY_FROM_SOURCE)
+  # Build and include hex file containing provisioned data for the bootloader.
+  set(PROVISION_HEX_NAME     provision.hex)
+  set(PROVISION_HEX          ${PROJECT_BINARY_DIR}/${PROVISION_HEX_NAME})
+  set(GENERATE_PROVISION_HEX TRUE)
+  set(PROVISION_HEX_EXISTS   TRUE)
+endif()
```

Comment:
-1 on breaking sysbuild support, we need to stop making changing to child/parent images